### ISSUE TITLE
[Question]: NULL IN PHYSICAL DEVICES IOS 

### ISSUE THREAD

**Issue by ameetaujla** [fluttercommunity/plus_plugins#1834]
### What is your question?

Hello, 
Guys I am using the latest version and I have iPhone 14 plus on the latest version IOS.  
it's working fine on the ios emulator. but returns null on the physical devices. 

here is the log. 
[] NEHotspotNetwork nehelper sent invalid result code [1] for Wi-Fi information request

I have added wifi access capability and tried everything but nothings work on latest version of IOS.

please help. i am new to ios, 

Thanks

### Checklist before submitting a question

- [X] I Google'd a solution and I couldn't find it
- [X] I searched on StackOverflow for a solution and I couldn't find it
- [X] I read the README.md file of the plugin
- [X] I am using the latest version of the plugin
- [X] All dependencies are up to date with `flutter pub upgrade`
- [X] I did a `flutter clean`
- [X] I tried running the example project

**Comment by vbuberen** [comment]
Please provide a better description of the issue, starting from the plugin you use and what is `null` as currently it is hard to understand what the problem is.

However, I think that you are talking about `network_info_plus`. There is an issue on latest iOS versions as discussed in #1652

**Comment by ameetaujla** [comment]
sorry for the less info. 

yes, I was talking about the same issue. regarding the network_info_plus plugin. 

I am using only getWifiIP() method it's returning null, and here are the logs. 
[] NEHotspotNetwork nehelper sent invalid result code [1] for Wi-Fi information request

code : final wifiIP = await info.getWifiIP();

Thanks,

**Comment by ESRSOL** [comment]
guys any updates, please?

**Comment by juanjuliocorica** [comment]
Hello, I have the same problem and I still couldn't solve it. Now I am looking to implement UDP to find the devices in the client's network but there are parts of the application closely linked to knowing the IP that are difficult to resolve in a clear way for the client.
I hope they can find a solution soon.

Thank you!

**Comment by blondie63** [comment]
I've same problem ! :(

**Comment by juanjuliocorica** [comment]
Hello everyone.
I also continue with the error and unfortunately I couldn't wait any longer because we already had many cases with problems.
Finally we implement a broadcast message via UDP so that the ESP32 responds to us and then we can capture the IP and connect.
In our case, we were able to solve it and for clients with old products we gave them the option of loading the IP by hand or updating the ESP32 firmware to incorporate the UDP tracking option.
I hope this helps others move forward with a possible resolution.
Greetings
